roles/tendrl-ansible.tendrl-server/tasks/tendrl-monitoring-integration.yml

```diff
@@ -80,6 +80,17 @@
   notify:
     - restart grafana-server
 
+- name: Configure grafana root_url with sub_path for reverse proxy
+  lineinfile:
+    dest=/etc/tendrl/monitoring-integration/grafana/grafana.ini
+    regexp={{ item.regexp }}
+    line={{ item.line }}
+  with_items:
+    - regexp: '^;? *root_url = .*'
+      line: "root_url = http://localhost:3000/grafana"
+  notify:
+    - restart grafana-server
+
```

Comment:
modifying:
;root_url = http://localhost:3000 to root_url = http://localhost:3000/grafana

Reply:
uncomment and adding subpath